### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by skumbananaaa**
The title says it all

### CODE REVIEW

**Review by Mumsfilibaba — APPROVED** [review]
I like this, very nice 👍

File: Assets/RenderGraphs/DEFERRED_PBR.lrg [hunk 1]
```diff
@@ -307,8 +307,8 @@
                 "texture_type": "TEXTURE_CUBE",
                 "x_dim_type": "CONSTANT",
                 "y_dim_type": "CONSTANT",
-                "x_dim_var": 512.0,
-                "y_dim_var": 512.0,
+                "x_dim_var": 1024.0,
+                "y_dim_var": 1024.0,
```
Comment (Ceanze) [inline Assets/RenderGraphs/DEFERRED_PBR.lrg]: Is it necessary to have increased size pushed to master?

**Review by Ceanze — CHANGES_REQUESTED** [review]

File: Assets/RenderGraphs/DEFERRED_PBR.lrg (hunk 1 above)
Comment (Mumsfilibaba) [reply Assets/RenderGraphs/DEFERRED_PBR.lrg]: Yes looks very nice 👍

Comment (Ceanze) [reply Assets/RenderGraphs/DEFERRED_PBR.lrg]: Ok very good 👍

**Review by Ceanze — APPROVED** [review]

**Pull request merged**